spec/unit/resources/osl_nextcloud_spec.rb

```diff
@@ -123,23 +123,43 @@
         end
 
         it do
+          packages = if platform[:version].to_i >= 9
+                       %w(
+                         bcmath
+                         gd
+                         gmp
+                         imagick
+                         intl
+                         json
+                         ldap
+                         mbstring
+                         mysqlnd
+                         opcache
+                         pecl-apcu
+                         pecl-redis6
+                         zip
+                       )
+                     else
+                       %w(
+                         bcmath
+                         gd
+                         gmp
+                         intl
+                         json
+                         ldap
+                         mbstring
+                         mysqlnd
+                         opcache
+                         pecl-apcu
+                         pecl-imagick
+                         pecl-redis6
+                         zip
+                       )
+                     end
```

Comment:
Let's shift this over like this:

```suggestion
          packages =
            if platform[:version].to_i >= 9
              %w(
                bcmath
                gd
                gmp
                imagick
                intl
                json
                ldap
                mbstring
                mysqlnd
                opcache
                pecl-apcu
                pecl-redis6
                zip
              )
            else
              %w(
                bcmath
                gd
                gmp
                intl
                json
                ldap
                mbstring
                mysqlnd
                opcache
                pecl-apcu
                pecl-imagick
                pecl-redis6
                zip
              )
            end
```